specifyweb/frontend/js_src/lib/components/QueryBuilder/Line.tsx

```diff
@@ -231,10 +239,19 @@ export function QueryLine({
 
   const hasAny = field.filters.some(({ type }) => type === 'any');
 
+  const fieldSpec = QueryFieldSpec.fromPath(baseTableName, field.mappingPath);
+  const rowTableName = generateMappingPathPreview(
+    fieldSpec.baseTable.name,
+    fieldSpec.toMappingPath()
+  );
+
+  const isBasic = React.useContext(IsQueryBasicContext);
+
   return (
     <div
       aria-current={isFocused ? 'location' : undefined}
-      className="flex flex-1 gap-2 border-t border-t-gray-500 bg-[color:var(--form-foreground)] py-2"
+      className={`flex flex-1 gap-2 border-t border-t-gray-500 bg-[color:var(--form-foreground)] py-2
```

Comment:
please insert newlines after first \` and before last \` to make it more readable